lib/contracts/message-reaction.ts

```diff
@@ -0,0 +1,36 @@
+import type { ContractDefinition } from '@balena/jellyfish-types/build/core';
+import { contractMixins } from 'autumndb';
+
+export const messageReaction: ContractDefinition = {
+	slug: 'message-reaction',
```

Comment:
We should probably decouple the concept of a reaction from the message contract. *Most* of the time they may be used with message contracts, but they will defintely be used with whispers as well, along with others we might not be thinking about right now. If we just have a `reaction` typed contract for example, it will be much more flexible and useful.